### PR TITLE
Add projects for GSOC 2022 for Keylime

### PR DESCRIPTION
**Pull request by mpeters**

### CODE REVIEW

**Review by nate-double-u — APPROVED** [review]

File: summerofcode/2022.md [hunk 1]
```diff
@@ -333,3 +333,33 @@ Description: The [in-toto Jenkins plugin](https://github.com/jenkinsci/in-toto-p
...
+### Keylime
+
+#### Support push model for agent attestation
+
+- Description: [Keylime](http://keylime.dev) enables users to monitor remote nodes (file integrity and measured boot) using a hardware based cryptographic root of trust. Keylime currently operates on a pull basis which means that the tenant or verifier connect to the agent to collect attestation data. This works fine in most virtualized environments where all the devices are in the same network, but not for edge devices or in BYOD contexts. This work would allow remote nodes to work in a "push" model instead of the normal "pull" model.
+- Recommended Skills: Python, Security
+- Mentor(s): Thore Sommer (@THS-on), Michael Peters (@mpeters), Marcio Silva (@maugustosilva)
+- Upstream Issue (URL): https://github.com/keylime/enhancements/issues/60
+
+#### Support push model for agent attestation
```
Comment (kkaarreell) [inline summerofcode/2022.md:346]: @mpeters Is it intentional that the topic name is identical to the previous one?

Comment (mpeters) [reply summerofcode/2022.md:346]: Good catch. New PR to fix it

**Comment by nate-double-u** [comment]
Hello,

We've been asked to provide some more information about the projects. 

As per the [Defining a Project Idea list](https://google.github.io/gsocguides/mentor/defining-a-project-ideas-list), could you please update any project suggestions you've submitted with the following (if not already provided):

a) a project title/description 
b) more detailed description of the project (2-5+ sentences) 
c) expected outcomes 
d) skills required/preferred 
e) possible mentors 
f) expected size of the project (175 or 350 hours) 
g) an easy, medium, or hard difficulty rating of each project.

Please update your projects by March 3, 2022. Thanks!

**Comment by nate-double-u** [comment]
Please see the umbrella issue: https://github.com/cncf/mentoring/issues/523

Note we have a new deadline: **Tues, March 1 at 5pm PT**.

**Pull request merged**